### ISSUE TITLE
A11y Guidance content

### ISSUE THREAD

**Issue by artuoma** [Office-of-Digital-Services/California-State-Web-Template-Website#1289]
Re-asses accessibility guidance on each component, possibly remove and consolidate into one area (Accessibility page?).

Create content on what tools and processes we user and/or should be used to test for accessibility with an emphasis on manual testing.
